github.cabal

```diff
@@ -1,7 +1,6 @@
 cabal-version:      >=1.10
 name:               github
-version:            0.27
-x-revision:         1
+version:            0.28
```

Comment:
Where is a breaking  change?

Reply:
No breaking change, only API addition.  So 0.27.1 should be fine.